english/trapezoid/train-ride/train-ride.md

```diff
@@ -0,0 +1,16 @@
+You are standing in front of a fire breathing, steam spitting locomotive.
+The Engineer grabs you by the collar and whisks you into the cab. 
+
+"Grab that shovel and start loading the fire with coal!", he bellows at you.
+"Keep your eye on the water gauge and work the injector! All right? Let's go!" 
+
+The Engineer hauls back on the throttle, gives a blast on the whistle and
+before you know it, you're feeding the beast with black gold as it accelerates
+in a cloud of smoke and steam vapor, rolling along the country side flowing
+past in a blur of speed.
+
+Hogwarts castle is in sight! It won't be long now! You have been waiting for this day your whole life.
+
+You hope your experience will be different from [others...](../magic/magic.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.